Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: test
+test:
+	nosetests
```

Comment:
@detiber @tbielawa when we move utils/Makefile into the top-level, we can have `make test` be the one to give very quick feedback during development, while keeping the `make ci` target doing all the things it does today (setup venv, unit tests, flake8, pylint).